edge/pkg/edged/apis/types.go

```diff
@@ -18,9 +21,39 @@ const (
 	StatusTag = "StatusTag"
 )
 
+// ContainerState is container state
+type ContainerState int32
+
+// Constants to show container status
+const (
+	ContainerCreated ContainerState = 0
+	ContainerRunning ContainerState = 1
+	ContainerExited  ContainerState = 2
+	ContainerUnknow  ContainerState = 3
+	ContainerPaused  ContainerState = 4
+)
+
+// Constants to check status of container state
+const (
+	StatusUNKNOWN                              = kubecontainer.ContainerStateUnknown
```

Comment:
StatusUNKNOWN  -> StatusUnknown.
Similarly for other constants.

Reply:
renamed these constants